lib/Cake/Utility/File.php

```diff
@@ -581,10 +581,10 @@ public function mime() {
  */
 	public function clearStatCache($all = false) {
 		if ($all === false && version_compare(PHP_VERSION, '5.3.0') >= 0) {
-			return clearstatcache(true, $this->path);
+			clearstatcache(true, $this->path);
```

Comment:
This change would make `clearstatcache()` be called twice.

Reply:
This specific code change can be kept now IMO.

Reply:
Me too.  but, This time, modify the docblock.